sopel/cli/run.py

```diff
@@ -255,11 +255,13 @@ def get_configuration(options):
     try:
         settings = utils.load_settings(options)
     except config.ConfigurationNotFound as error:
-        print(
-            "Welcome to Sopel!\n"
-            "I can't seem to find the configuration file, "
-            "so let's generate it!\n")
-        settings = utils.wizard(error.filename)
+        raise config.ConfigurationError(
+            str(error) + (
+                "\n"
+                "If you're just setting up Sopel, welcome! "
+                "You can use `sopel configure` to get started easily."
+            )
+        )
```

Comment:
No more extra parentheses with `%` (you can replace that with `.format()` if you prefer):

```suggestion
        raise config.ConfigurationError(
            "%s\n"
            "If you're just setting up Sopel, welcome! "
            "You can use `sopel configure` to get started easily." % error
        )
```

Reply:
I'll just make it an f-string :smile_cat: